parser-typechecker/src/Unison/Runtime/Interface.hs

```diff
@@ -490,21 +490,22 @@ decodeStandalone b = bimap thd thd $ runGetOrFail g b
         <*> getNat
         <*> getStoredCache
 
--- | Whether the runtime is hosted within a UCM session or as a standalone process.
+-- | Whether the runtime is hosted within a persistent session or as a one-off process.
+-- This affects the amount of clean-up and book-keeping the runtime does.
 data RuntimeHost
-  = Standalone
-  | UCM
```

Comment:
"UCM" mode didn't make sense on the Enlil server

Reply:
What do OneOff and Persistent mean? What's a one-off process?

Reply:
If you use `unison run ...` then ucm spins up the runtime, runs the given 'main', then immediately shuts down; as opposed to a "persistent" ucm session which may run many different code snippets, evaluate docs, etc.

The difference is that the one-off version doesn't track or clean up any forked threads since they're killed by the OS upon completion anyways, whereas if you run a server in UCM and then ctrl-c, the runtime needs to clean up any background threads or your server will keep running indefinitely.

Reply:
Ah ok. So... `unison run` and `unison transcript` are one-off, and `unison` and `unison headless` are persistent?